packages/bot-web-ui/src/components/toolbar.jsx

```diff
@@ -107,7 +107,7 @@ const BotNameBox = ({ onBotNameTyped, file_name }) => (
     </div>
 );
 
-const popover_zindex = config.popover_zindex.toolbar;
+const popover_zindex = Number(config.popover_zindex.toolbar);
```

Comment:
Same here, no need to typecast.

```suggestion
const popover_zindex = config.popover_zindex.toolbar;
```